bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/resources/OH-INF/i18n/helioseasycontrols_de.properties

```
@@ -0,0 +1,204 @@
+channel-group-type.modbus.vocControl.label = VOC-Steuerung
+channel-group-type.modbus.vocControl.description = Konfiguration der Sensoren f�r die VOC-Steuerung
+
+# channel types
+channel-type.modbus.sysdate.label = Systemdatum und -uhrzeit
+channel-type.modbus.sysdate.description = Das Systemdatum bzw. die Systemuhrzeit der KWL 
+channel-type.modbus.summerWinter.label = Sommer-/Winterzeit
+channel-type.modbus.summerWinter.description = Zeigt an, ob Sommer- oder Winterzeit aktiv ist
+channel-type.modbus.summerWinter.state.option.0 = Winterzeit
+channel-type.modbus.summerWinter.state.option.1 = Sommerzeit
+channel-type.modbus.enableDisable.label = Aktivieren/deaktivieren 
+channel-type.modbus.enableDisable.description = Verwendet f�r Funktionalit�t die aktiviert oder deaktiviert sein kann
+channel-type.modbus.minFanStage.label = Minimale L�fterstufe
+channel-type.modbus.minFanStage.description =  Minimale L�fterstufe (0 oder 1)
+channel-type.modbus.onOff.label = Ein/aus
+channel-type.modbus.onOff.description = Verwendet f�r Konfigurationen die ein- oder ausgeschaltet sein k�nnen
+channel-type.modbus.humidityControlSetValue.label = Feuchte-Steuerung Sollwert
+channel-type.modbus.humidityControlSetValue.description = Feuchte-Steuerung Sollwert (in Prozent)
+channel-type.modbus.humidityControlSteps.label = Feuchte-Steuerung Stufen
+channel-type.modbus.humidityControlSteps.description = Feuchte-Steuerung Stufen (in Prozent)
+channel-type.modbus.humidityStopTime.label = Feuchte-Steuerung Stoppzeit
+channel-type.modbus.humidityStopTime.description = Feuchte-Steuerung Stoppzeit in Stunden (0-24)
+channel-type.modbus.ppmControlSetValue.label = Sollwert ppm-Steuerung 
+channel-type.modbus.ppmControlSetValue.description = Sollwert f�r ppm-basierte Steuerung (in ppm)
+channel-type.modbus.ppmControlSteps.label = Stufen ppm-Steuerung
+channel-type.modbus.ppmControlSteps.description = Stufen f�r ppm-basierte Steuerung (in ppm)
+channel-type.modbus.comfortTemp.label = Komforttemperatur
+channel-type.modbus.comfortTemp.description = Komforttemperatur 
+channel-type.modbus.duration.label = Dauer
+channel-type.modbus.duration.description = Dauer f�r Betriebsarten (in Minuten)
+channel-type.modbus.fanStage.label = L�fterstufe
+channel-type.modbus.fanStage.description = L�fterstufe 
+channel-type.modbus.remainingTime.label = Restzeit
+channel-type.modbus.remainingTime.description = Restzeit f�r Betriebsarten (in Minuten)
+channel-type.modbus.operatingMode.label = Betriebsart
+channel-type.modbus.operatingMode.description = Betriebsart (automatisch/manuell)
+channel-type.modbus.operatingMode.state.option.0 = automatisch
+channel-type.modbus.operatingMode.state.option.1 = manuell
+channel-type.modbus.percentage.label = Prozentwert
+channel-type.modbus.percentage.description = Prozentwert 
+channel-type.modbus.temperature.label = Temperatur
+channel-type.modbus.temperature.description = Temperatur in �C 
```

The user could select different units if you make use of Units of Measure.
```suggestion
channel-type.modbus.temperature.description = Temperatur
```